packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-authz-server/index.md

```diff
@@ -2,15 +2,14 @@
 title: Create an authorization server
 ---
 
-> **Note:** If you have an [Okta Developer Edition](https://developer.okta.com/signup/) account and you don't want to create any additional custom authorization servers, you can skip this step because you already have a custom authorization server created for you called "default". The `{authServerId}` for the default server is `default`.
+> **Note:** If you have an [Okta Developer Edition](https://developer.okta.com/signup/) account and you don't want to create any additional Custom Authorization Servers, you can skip this step because you already have a Custom Authorization Server created for you called "default". The `{authServerId}` for the default server is `default`.
 
 1. In the Developer Console, navigate to **API > Authorization Servers**.
 
-2. Select **Add Authorization Server** and enter the **Name**, **Audience**, and **Description** for the authorization server.
+2. Select **Add Authorization Server** and enter the **Name**, **Audience**, and **Description** for the Authorization Server.
 
-  > **Note:** The **Audience** is the URI for the OAuth 2.0 resource server that consumes the access tokens. Use an absolute path such as `https://api.example.com/pets`.
-      This value is used as the default [audience](https://tools.ietf.org/html/rfc7519#section-4.1.3) for access tokens.
+  > **Note:** An access token that is minted by a Custom Authorization Server requires that you define the **Audience** (`aud` parameter). The **Audience** property should be set to the URI for the OAuth 2.0 resource server that consumes the access token. When validating the access token, the resource server rejects an access token if the `aud` claim isn't set to the **Audience** value that you configure in the Custom Authorization Server. Use an absolute path such as `https://api.example.com/pets`. This value is used as the default [audience](https://tools.ietf.org/html/rfc7519#section-4.1.3) for access tokens.
```

Comment:
Is the `aud` a parameter or a claim? This might be confusing as we call it different things in this paragraph.
Will slack you on suggested re-write